### ISSUE TITLE
About setting the hyperparameters

### ISSUE THREAD

**Issue by OuyKai** [tk-rusch/coRNN#2]
How are the parameters dt，gamma，and epsilon of coRNN determined？ Are there any tips for setting the hyperparameters?

**Comment by tk-rusch** [comment]
We did a simple random search to find good hyperparameters dt, gamma and epsilon. If you don't want to fine-tune all three, fixing both gamma and epsilon to 1 and fine-tuning the time-step dt between 0.001 and 1 (depending on the data and length of the sequences) should already give very good results. 
Moreover, if you look at our paper, we actually did an ablation study regarding gamma and epsilon and found that the model is much more sensitive to gamma than to epsilon, which can be seen also theoretically by looking at Proposition F.2 and F.3. There, epsilon comes in only indirectly through a constraint on the time step , while gamma has a direct influence on the gradients and hidden states.
Does that answer your questions?
